### PR TITLE
Add new properties to the FabricLocation resource

### PR DESCRIPTION
**Pull request by cednation**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json [hunk 1]
```diff
@@ -112,6 +113,63 @@
...
+        "type": "object",
+        "properties": {
+          "pepIpAddresses": {
+            "description": "The IP addresses of the privileged endpoints.",
+            "type": "array",
+            "items": {
+              "type": "string"
+        }
+      },
+      "timeServer": {
```
Comment (raych1) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: Can you fix the indent?

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json [hunk 2]
```diff
@@ -97,13 +97,14 @@
     }
   },
   "definitions": {
-    "FabricLocation": {
-      "description": "Place holder for fabric location.",
+   "FabricLocation": {
```
Comment (raych1) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: Is this model not being used before?

Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: Yes. Previously the FabricLocation had no properties. In recent changes we have added properties to the resource.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json (hunk 1 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json]: Yes, of course. Fixed.

**Review by raych1 — APPROVED** [review]
LGTM.

**Pull request merged**